src/main/java/org/assertj/core/api/AbstractFileAssert.java

```diff
@@ -1234,4 +1234,27 @@ public SELF isNotEmpty() {
     files.assertIsNotEmptyFile(info, actual);
     return myself;
   }
+
+  /**
+   * Get a StringAssert object with the content of the actual {@code File}.
+   * <p>
+   * Example:
+   * <pre><code class='java'> File file = File.createTempFile(&quot;tmp&quot;, &quot;txt&quot;);
+   * Files.write(file.toPath(), new byte[]{'t', 'e', 's', 't'});
+   *
+   * // assertion will pass
+   * assertThat(file).content().contains('te');
+   *
+   * // assertion will fail
+   * assertThat(file).content().contains('tet');</code></pre>
+   *
+   * @return a StringAssert object.
+   * @throws AssertionError if the actual {@code File} is not readable.
+   * @since 3.??
+   */
+  public StringAssert content() {
+    String fileContent = files.getFileContent(info, actual);
+    return new StringAssert(fileContent);
+  }
```

Comment:
we need to add `public StringAssert content(Charset actualCharset)` and make this one use the default charset.